.gitignore

```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
```

Comment:
I'm not sure this file should be added to the project, it seems specific to your IDE

Reply:
Even though it seems that this does not belong to the general project, we can get rid of it later if need be.